test/migration_generator_test.exs

```diff
@@ -77,44 +83,30 @@ defmodule AshPostgres.MigrationGeneratorTest do
       :ok
     end
 
-    test "it creates a snapshot for each resource" do
```

Comment:
I consolidated all of these tests into one as the migration_generator takes a bit to run and these were all checking off the same result